src/internal.c

```diff
@@ -65,6 +65,12 @@
  *     may be received by a client. To support detecting this, peek will
  *     return WOLFSSL_ERROR_WANT_READ.
  *     This define turns off this behaviour.
+ * WOLFSSL_FORCE_AUTO_RETRY
```

Comment:
So defining this macro forces the network reads to behave as blocking? Is there a reason not to implement runtime support using the `SSL_MODE_AUTO_RETRY` option? Also can you check review the code comment at ssl.h:2136 for `SSL_MODE_AUTO_RETRY`? It seems wrong.

```
SSL_MODE_AUTO_RETRY = 3, /* wolfSSL default is to block with blocking io and auto retry */
```

Reply:
I wanted to keep this PR minimum but if you would prefer a runtime solution, I can add the necessary API to make that work. Still, I don't think that this is a good default for wolfSSL. If you want me to implement a runtime solution, I'll make the `SSL_MODE_AUTO_RETRY` option on by default only with `OPENSSL_COMPATIBLE_DEFAULTS`.

Reply:
I'll update the `SSL_MODE_AUTO_RETRY` comment to match the real behaviour.

Reply:
Comment fixed.